app/pages/items/description/template.njk

```diff
@@ -0,0 +1,31 @@
+{% extends 'includes/layout.njk' %}
+{% from 'components/back-link/macro.njk' import backLink %}
+{% from 'components/bc-button/macro.njk' import bcButton %}
+{% from 'components/textarea-field-no-count/macro.njk' import textareaFieldNoCount %}
+
+{% block body %}
+  <div class="nhsuk-width-container" data-test-id="description-page">
+    <div data-test-id="go-back-link">
+      {{ backLink({
+        "href": backlinkHref,
+        "text": backlinkText
+      }) }}
+    </div>
+
+    <div class="nhsuk-grid-row">
+      <div class="nhsuk-grid-column-two-thirds">
+        <h1 data-test-id="description-page-title" class="nhsuk-u-margin-top-7 nhsuk-u-margin-bottom-3">{{ title }}</h1>
+        <h2 data-test-id="description-page-description" class="nhsuk-u-font-size-24 nhsuk-u-margin-bottom-6">{{ description }}</h2>
+
+        {{ textareaFieldNoCount(descriptionQuestion) }}
+
+        <div data-test-id="save-button">
```

Comment:
you can pass this dataTestId directly to the component and we don't need the this div then